app/api/v1/auth.py

```
@@ -72,10 +62,30 @@ async def signup(
-
     except TicketInvalidError as e:
         raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_CONTENT, detail=str(e))
     except DuplicateEmailError as e:
-        raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail=str(e))
\ No newline at end of file
+        raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail=str(e))
+
+@router.post("/login", response_model=Token)
+async def login(
+    payload: LoginRequest,
+    uc: AuthenticateUser = Depends(get_authenticate_user_uc),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)